javascript-loops/main.js

```diff
@@ -0,0 +1,72 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+
+  while (count <= times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i=0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (i = 0; i < numbers.length; i++) {
```

Comment:
always declare your variables in a for loop! Otherwise they become auto globals, and that is bad news!